common/app/views/support/Profile.scala

```diff
@@ -96,25 +96,32 @@ object Item640 extends Profile(width = Some(640))
 object Item700 extends Profile(width = Some(700))
 object Video640 extends VideoProfile(width = Some(640), height = Some(360)) // 16:9
 object Video700 extends VideoProfile(width = Some(700), height = Some(394)) // 16:9
-object TwitterImage extends Profile(width = Some(1200))
-object FacebookOpenGraphImage extends Profile(width = Some(1200)) {
 
-  override val heightParam = "h=632"
+class ShareImage(shouldIncludeOverlay: Boolean) extends Profile(width = Some(1200)) {
```

Comment:
it may be worth making this a `trait` and declaring `blendImageParam` abstract ie `blendImageParam: String;` rather than just making it an empty string. This would force all classes that extend `ShareImage` to proved a `blendImageParam`